### ISSUE TITLE
jline prints a CRLF on macOS and Linux for a new line

### ISSUE THREAD

**Issue by chrisseaton** [jline/jline2#282]
Console reader's `println` is

```java
    public void println() throws IOException {
        tputs("carriage_return");
        rawPrint('\n');
    }
```

`UnixTerminal` has `carriage_return` as `^M`, which is `CR`, and then you `rawPrint` the `LF`.

Isn't `CRLF` a Windows thing? Why do you print it here? Is `UnixTerminal` getting confused somehow? I'm redirecting `stdout` to a file in my shell if that makes a difference.

I've seen this happening on macOS and Linux, but on some colleagues Linux machines they seem to be getting just `LF`.

**Comment by eregon** [comment]
This is a regression between JLine 2.11 (working) and JLine 2.14.3 which contains this bug.

This is the reason why some Linux machines behaved as expected: they were using the older JLine.

**Comment by gnodet** [comment]
I think it should use the System.lineSeparator() instead.
Could someone provide a PR ?

**Comment by eregon** [comment]
Bisecting reveals commit 3b0c474af7b46245109487083f1c0a0fd053165e introduced the extra `\r`.
I'll try to make a PR.

**Comment by eregon** [comment]
PR created, with test and ensuring it fails before and succeeds after the fix: #283 .
